resolwe_bio/processes/expression/slamdunk.py

```diff
@@ -0,0 +1,105 @@
+import os
+
+from plumbum import TEE
+
+from resolwe.process import Process, Cmd, DataField, BooleanField, IntegerField, FileField, DirField
+
+
+class SlamdunkAllPaired(Process):
+    """Run slamdunk for paired-end reads with alignment to transcriptome."""
```

Comment:
I would avoid the `transcriptome` notation here, since its possible to run the tool against any nucleotide sequence + bed regions file (as in the case of QUANT-Seq data).